### ISSUE TITLE
question about the function shift

### ISSUE THREAD

**Issue by mvalitov** [bitwalker/timex#377]
### Steps to reproduce

```
iex(4)> {:ok, dt} = Timex.parse("2017-08-31 21:50:49.000000Z", "{ISO:Extended}")
{:ok, #DateTime<2017-08-31 21:50:49.000000Z>}
iex(5)> dt2 = Timex.shift(dt, months: 1)
#DateTime<2017-10-01 21:50:49.000000Z>
iex(10)> Timex.shift(dt2, months: -1)    
#DateTime<2017-09-01 21:50:49.000000Z>

```

### Description of issue

- What are the expected results?
`2017-09-30 21:50:49`

maybe I misunderstand how the shift should work, but why the result is 2017-10-01 21:50:49, not 2017-09-30 21:50:49?

**Comment by tmaszk** [comment]
I noticed the same thing and raised issue https://github.com/bitwalker/timex/issues/360

I have submitted a PR to resolve it.  Hopefully it will get merged.

In the meantime, I believe you will get the result you expect if you use a Native Date structure instead of DateTime.

**Comment by bitwalker** [comment]
Closed via #360
